server/src/test/java/org/elasticsearch/indices/IndicesRequestCacheIT.java

```diff
@@ -243,26 +245,21 @@ public void testQueryRewriteDatesWithNow() throws Exception {
         assertAcked(client.admin().indices().prepareCreate("index-3").addMapping("type", "d", "type=date")
                 .setSettings(settings).get());
         ZonedDateTime now = ZonedDateTime.now(ZoneOffset.UTC);
-        indexRandom(true, client.prepareIndex("index-1", "type", "1").setSource("d", now),
-                client.prepareIndex("index-1", "type", "2").setSource("d", now.minusDays(1)),
-                client.prepareIndex("index-1", "type", "3").setSource("d", now.minusDays(2)),
-                client.prepareIndex("index-2", "type", "4").setSource("d", now.minusDays(3)),
-                client.prepareIndex("index-2", "type", "5").setSource("d", now.minusDays(4)),
-                client.prepareIndex("index-2", "type", "6").setSource("d", now.minusDays(5)),
-                client.prepareIndex("index-3", "type", "7").setSource("d", now.minusDays(6)),
-                client.prepareIndex("index-3", "type", "8").setSource("d", now.minusDays(7)),
-                client.prepareIndex("index-3", "type", "9").setSource("d", now.minusDays(8)));
-        ensureSearchable("index-1", "index-2", "index-3");
-        assertCacheState(client, "index-1", 0, 0);
-        assertCacheState(client, "index-2", 0, 0);
-        assertCacheState(client, "index-3", 0, 0);
 
-        // Force merge the index to ensure there can be no background merges during the subsequent searches that would invalidate the cache
-        ForceMergeResponse forceMergeResponse = client.admin().indices().prepareForceMerge("index-1", "index-2", "index-3").setFlush(true)
-                .get();
-        ElasticsearchAssertions.assertAllSuccessful(forceMergeResponse);
-        refresh();
+        BulkRequestBuilder bulkBuilder = client().prepareBulk();
+        bulkBuilder.add(client.prepareIndex("index-1", "type", "1").setSource("d", now));
+        bulkBuilder.add(client.prepareIndex("index-1", "type", "2").setSource("d", now.minusDays(1)));
+        bulkBuilder.add(client.prepareIndex("index-1", "type", "3").setSource("d", now.minusDays(2)));
+        bulkBuilder.add(client.prepareIndex("index-2", "type", "4").setSource("d", now.minusDays(3)));
+        bulkBuilder.add(client.prepareIndex("index-2", "type", "5").setSource("d", now.minusDays(4)));
+        bulkBuilder.add(client.prepareIndex("index-2", "type", "6").setSource("d", now.minusDays(5)));
+        bulkBuilder.add(client.prepareIndex("index-3", "type", "7").setSource("d", now.minusDays(6)));
+        bulkBuilder.add(client.prepareIndex("index-3", "type", "8").setSource("d", now.minusDays(7)));
+        bulkBuilder.add(client.prepareIndex("index-3", "type", "9").setSource("d", now.minusDays(8)));
+        BulkResponse actionGet = bulkBuilder.execute().actionGet();
```

Comment:
same here

Reply:
Thanks @dnhatn, addressed in 7f125e75efd8f697a6fae8820e9712ed8b36a25d